src/pages/sermons/videoComp.tsx

```diff
@@ -0,0 +1,168 @@
+/* eslint-disable @next/next/no-img-element */
+import Link from "next/link";
+
+const VideoComp = () => {
```

Comment:
move this component to `/components/sermons/VideoComp.tsx`. Anything inside `/pages` is a page and the videoComp isn't a page.